tooling/api/src/main/java/org/jboss/windup/tooling/data/Classification.java

```diff
@@ -39,6 +39,12 @@
     Iterable<Link> getLinks();
 
     /**
+     * This contains a list of {@link Quickfix}s for follow up in tools
+     * @return
+     */
+    Iterable<Quickfix> getQuickfixes();
```

Comment:
Do you mean the multiple quickfixes or just quickfixes? I thought it is a rare case, but why not.

Actually I wasn't too convinced, but classification was in planning document.